microcosm_pubsub/daemon.py

```diff
@@ -16,34 +14,36 @@ def make_arg_parser(self):
         parser.add_argument("--sqs-queue-url")
         return parser
 
-    @abstractproperty
     def schema_mappings(self):
         """
         Define the PubSub message media-type to schema mappings.
 
         """
-        pass
+        return dict()
+
+    def create_object_graph(self, args):
+        graph = super(ConsumerDaemon, self).create_object_graph(args)
+        for media_type, schema_cls in self.schema_mappings.items():
+            self.graph.pubsub_message_schema_registry.register(media_type, schema_cls)
+        return graph
 
     @property
     def defaults(self):
-        dct = dict(
-            pubsub_message_codecs=dict(
```

Comment:
I'm confused. 
Aren't `pubsub_message_codecs` the messages we want to **publish**? How's that related to this PR?